### PR TITLE
Assignments fe base implementation added

### PR DESCRIPTION
**Pull request by beasterbro**
Implementing assignments page and all of its basic functionality

### CODE REVIEW

File: services/frontend/src/components/assignments/assignments-page.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        this.didSelectAssignment = this.didSelectAssignment.bind(this);
+
+        this.state = {
+            selectedAid: undefined,
+        };
+    }
+
+    componentDidMount() {
+        api.setBearerToken(
+            'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiJmYWJiMDQyZi05NjU2LTRjMjAtYmYzMy1hZmM5MDMzN2E1ZTEiLCJpYXQiOjE2MDE4NDM3OTB9.oaMx3ATdIOYikkdMPI4f8lnAIcS0z5hAaP6hODOQUC8'
```
Comment (prao3) [inline services/frontend/src/components/assignments/assignments-page.js:23]: You shouldn't need to do this anymore, you should be able to just login and have a user specific bearer token

File: services/frontend/gatsby-browser.js [hunk 2]
```diff
@@ -1,3 +1 @@
-import './src/globals/global.scss';
-
-export { default as wrapRootElement } from './src/globals/RootWrapper';
+export { default as wrapRootElement } from './src/store/ReduxWrapper';
```
Comment (lucaspopp0) [inline services/frontend/gatsby-browser.js:1]: Bad change cuz the PR is old

File: services/frontend/gatsby-ssr.js [hunk 3]
```diff
@@ -1,3 +1 @@
-import './src/globals/global.scss';
-
-export { default as wrapRootElement } from './src/globals/RootWrapper';
+export { default as wrapRootElement } from './src/store/ReduxWrapper';
```
Comment (lucaspopp0) [inline services/frontend/gatsby-ssr.js:1]: Another outdated change

File: services/frontend/src/components/assignments/assignment-cell.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+        this.props.updateAssignmentLocal({
+            id: this.props.aid,
+            changes,
+        });
+
+        this.setState({
+            assignment: _.merge({}, this.state.assignment, changes),
+        });
+    }
```
Comment (lucaspopp0) [inline services/frontend/src/components/assignments/assignment-cell.js:48]: I don't think this method is used

File: services/frontend/src/components/tasks/tasks-page.js [hunk 5]
```diff
@@ -16,6 +17,9 @@ class TasksPage extends React.Component {
     }
 
     componentDidMount() {
+        api.setBearerToken(
+            'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiJmYWJiMDQyZi05NjU2LTRjMjAtYmYzMy1hZmM5MDMzN2E1ZTEiLCJpYXQiOjE2MDE4NDM3OTB9.oaMx3ATdIOYikkdMPI4f8lnAIcS0z5hAaP6hODOQUC8'
+        );
```
Comment (lucaspopp0) [inline services/frontend/src/components/tasks/tasks-page.js:22]: Also not needed anymore

File: services/frontend/src/pages/tasks.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import TasksPage from '../components/tasks/tasks-page';
 
-export default function wrapper() {
-    return <TasksPage />;
+export default function Tasks() {
+    return <TasksPage></TasksPage>;
```
Comment (lucaspopp0) [inline services/frontend/src/pages/tasks.js:5]: Bad change from the merge?

File: services/frontend/src/store/ReduxWrapper.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import store from './index';
+
+function wrapper({ element }) {
+    return <Provider store={store}>{element}</Provider>;
+}
```
Comment (lucaspopp0) [inline services/frontend/src/store/ReduxWrapper.js:7]: This got deleted a while ago

File: services/frontend/src/styles/assignments.css [hunk 8]
```diff
@@ -0,0 +1,3 @@
+#h3{
```
Comment (lucaspopp0) [inline services/frontend/src/styles/assignments.css:1]: Delete

File: services/frontend/src/pages/tasks.js (hunk 6 above)
Comment (beasterbro) [reply services/frontend/src/pages/tasks.js:5]: This is a change that eslint requested or the test would fail

Comment (lucaspopp0) [reply services/frontend/src/pages/tasks.js:5]: Are you sure? It works on master just fine

File: services/frontend/src/components/assignments/assignments-page.js (hunk 1 above)
Comment (beasterbro) [reply services/frontend/src/components/assignments/assignments-page.js:23]: done

File: services/frontend/src/components/tasks/tasks-page.js (hunk 5 above)
Comment (beasterbro) [reply services/frontend/src/components/tasks/tasks-page.js:22]: done

File: services/frontend/src/pages/tasks.js (hunk 6 above)
Comment (beasterbro) [reply services/frontend/src/pages/tasks.js:5]: im sure, the test results on github pointed directly to this line, I'll try to revert it

Comment (beasterbro) [reply services/frontend/src/pages/tasks.js:5]: it seemed to have worked

File: services/frontend/gatsby-browser.js (hunk 2 above)
Comment (beasterbro) [reply services/frontend/gatsby-browser.js:1]: fixed

File: services/frontend/src/components/assignments/assignment-cell.js (hunk 4 above)
Comment (beasterbro) [reply services/frontend/src/components/assignments/assignment-cell.js:48]: it wasn't I removed it

File: services/frontend/src/styles/assignments.css (hunk 8 above)
Comment (beasterbro) [reply services/frontend/src/styles/assignments.css:1]: removed outdated file

File: services/frontend/src/store/ReduxWrapper.js (hunk 7 above)
Comment (beasterbro) [reply services/frontend/src/store/ReduxWrapper.js:7]: removed

File: services/frontend/gatsby-ssr.js (hunk 3 above)
Comment (beasterbro) [reply services/frontend/gatsby-ssr.js:1]: fixed

**Review by lucaspopp0 — APPROVED** [review]

**Pull request merged**